src/Rector/v8/v0/RemoveLangCsConvObjAndParserFactoryRector.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v8\v0;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Charset\CharsetConverter;
+use TYPO3\CMS\Core\Localization\LanguageService;
+use TYPO3\CMS\Core\Localization\LocalizationFactory;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/8.0/Deprecation-73482-LANG-csConvObjAndLANG-parserFactory.html
+ */
+final class RemoveLangCsConvObjAndParserFactoryRector extends AbstractRector
+{
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, LanguageService::class)) {
+            return null;
+        }
+
+        if ($this->isName($node->name, 'csConvObj')) {
```

Comment:
I have doing it wrong here. I have changed the function call of `csConvObj()`  and not the property usage.
I'll look into this again.

Reply:
Right. And we have to deal with the $GLOBALS['LANG'] here too. So you need the Typo3NodeResolver class and check if we are dealing with a propertyFetch on $GLOBALS. See i.e. https://github.com/sabbelasichon/typo3-rector/blob/master/src/Rector/v9/v5/RefactorPropertiesOfTypoScriptFrontendControllerRector.php#L43